<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>702</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>pyBind</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Submission</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Type of input</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputType">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Fasta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Peptide</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste a single sequence or several sequencies in FASTA format into the field below:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>or submit a file a file in Fasta format directly from your local disk:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose file</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>no file selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Peptide length (you may select multiple lengths)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="inputMers">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <item>
     <property name="text">
      <string>8mer peptides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9mer peptides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10mer peptides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11mer peptides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12mer peptides</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select loci</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputHLACategory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All HLAs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLA-A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLA-B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLA-C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLA-E</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select allele</string>
    </property>
   </widget>
   <widget class="QListWidget" name="inputHLAallele">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Threshold for strong binder: % Rank</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Threshold for weak binder: % Rank</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="thresholdStrong">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="thresholdWeak">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="runbtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>640</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run netMCHpan3.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
